src/backend/config.ts

```diff
@@ -26,6 +26,7 @@ import {
   getDefaultWine,
   getGamingPortingToolkitWine,
   getLinuxWineSet,
+  getSystemGamingPortingToolkitWine,
```

Comment:
any reason to name this `Gaming` porting toolkit instead of `Game` porting toolkit like in the rest?

Reply:
No reason, was automatic I guess.